### PR TITLE
feature: update footer and navLink  alignment

### PR DESCRIPTION
**Pull request by NayanUnni95**
## update footer and navLink  alignment
- fix navbar menu icon alignment(xs, sm, and md).
-  setup website icon loading issue.
- add more social media link.
- change footer alignment.

### CODE REVIEW

**Review by Geeksphere — APPROVED** [review]
pull request approved

**Comment by NayanUnni95** [comment]
#### resolve merge conflict

**Comment by Geeksphere** [comment]
the pull requst is merged

File: assets/js/main.js [hunk 1]
```diff
@@ -1,10 +1,3 @@
-/**
-* Template Name: Ninestars
-* Updated: Jan 29 2024 with Bootstrap v5.3.2
-* Template URL: https://bootstrapmade.com/ninestars-free-bootstrap-3-theme-for-creative/
```
Comment (Geeksphere) [inline assets/js/main.js:4]: removed some  definitions

**Review by Geeksphere — COMMENTED** [review]
added icons of social  handles in footer and adjust the aliment of nav-link

**Pull request merged**